jmeter-java-dsl/src/main/java/us/abstracta/jmeter/javadsl/JmeterDsl.java

```diff
@@ -622,6 +623,21 @@ public static DslForEachController forEachController(String name, String varsPre
   public static DslOnceOnlyController onceOnlyController(ThreadGroupChild... children) {
     return new DslOnceOnlyController(Arrays.asList(children));
   }
+  /**
+   * Builds a Runtime Controller that allows running a part of a test plan for a specified
```

Comment:
Maybe makes sense to specify when would you use it as a general advice to other users. And also describe what happens if the time reaches before the test plan part ends.

Reply:
I cAdded more detail